tool/tctl/common/access_request_command.go

```diff
@@ -264,7 +266,7 @@ func (c *AccessRequestCommand) Create(ctx context.Context, client auth.ClientI)
 	req.SetRequestReason(c.reason)
 
 	if c.dryRun {
-		err = services.ValidateAccessRequestForUser(ctx, client, req, services.ExpandVars(true))
+		err = services.ValidateAccessRequestForUser(ctx, clockwork.NewFakeClock(), client, req, tlsca.Identity{}, services.ExpandVars(true))
```

Comment:
Shouldn't this be a real clock?
```suggestion
		err = services.ValidateAccessRequestForUser(ctx, clockwork.NewRealClock(), client, req, tlsca.Identity{}, services.ExpandVars(true))
```